msg.py

```diff
@@ -111,25 +113,29 @@
         '{1}: Popp',
         '{1}: Pop',
         '{1}: Blub (Huch, der war wohl mit Wasser gefüllt.)',
+        '{1}: Puff (Wo kommt denn all die Luft her? Schäm dich, {0}, wenn du was anderes dachtest ;) )',
     ],
     'sehr_toll': [  # {0} is sender_firstname, {1} is sender_username
         'Ja!',
         '"immer mit ja" :D',
         'Aber natürlich, {0}!',
         'Du bist ganz toll, {0} ^^',
-        'Ich antworte dir weil du so toll bist :D',
-        'Sehr geehrter User @{1},\n\nhiermit möchten wir Ihnen bestätigen und zertifizieren, dass Sie für Tollheit Güteklasse 1 qualifizieren. Herzlichen Glückwunsch, und alles Gute auf ihren Tollkühnen Reisen!\n\nMit freundlichem Gruß,\nIhr @FidgetCubeBot',
+        'Ich antworte dir, weil du so toll bist :D',
+        'Sehr geehrter User @{1},\n\nhiermit möchten wir Ihnen bestätigen und zertifizieren, dass Sie sich für Tollheit der Güteklasse 1 qualifizieren. Herzlichen Glückwunsch, und alles Gute auf Ihren tollkühnen Reisen!\n\nMit freundlichem Gruß,\nIhr @FidgetCubeBot',
         'Na klaro!',
         'Ja sowas von!',
         'Jup :)',
         'Nein – du bist *sehr* toll! ;)',
         ':o Ich bin nur ein Bot und habe keine Arme, aber fühl dich gedrückt, {0}, du toller Mensch!',
         'Alle mal herhören! {0} ist ganz toll, ja?! So, das wäre dann ja wohl geklärt.',
+        '{0} ist toll toll toll toll toll, toller geht\'s nicht.',
     ],
     'unknown_command': [  # {0} is sender_firstname
         'Häh?',
         'Bittewas, {0}?',
         'Bestimmt weiß ich eines Tages, was das bedeuten soll, {0}.',
         'Ich habe nicht genügend Erfahrung, um diese Aufgabe auszuführen.',
+        'Watt willste?',
+        'Es ist so dunkel, ich kann dich nicht hören!',
```

Comment:
Wasn't the quote slightly different?

> Ich kann dich nicht hören, es ist so dunkel hier drinnen!

https://www.youtube.com/watch?v=8FAaiJqJHzs